packages/grid/x-data-grid/src/components/cell/GridEditSingleSelectCell.tsx

```diff
@@ -38,9 +38,11 @@ export interface GridEditSingleSelectCellProps
    * @returns {Promise<void> | void} A promise to be awaited before calling `apiRef.current.setEditCellValue`
    */
   onValueChange?: (event: SelectChangeEvent<any>, newValue: any) => Promise<void> | void;
+  initialOpen?: boolean;
```

Comment:
Could you fill the description? Once we have that it's good to be merged.

Reply:
circle_ci test_static was failing when I added description
<img width="352" alt="image" src="https://user-images.githubusercontent.com/48183855/182182713-73a3e290-4b3d-48c0-b7d8-c7d2b0f12e13.png">

Reply:
You need to run `yarn proptypes` and commit the result too. The proptypes are generated by reading the component's prop interface.

Reply:
Yeah I already ran `yarn proptyes` and then added comment over generated proptype for description.

Reply:
In the first commit I did add description above proptype as a comment that's when test_static was failing then I removed it

Reply:
I fixed the description.